src/main/java/hudson/plugins/promoted_builds/PromotionProcess.java

```diff
@@ -90,6 +91,11 @@
      */
     public String isVisible;
 
+    /**
+     * Trigger promotion when any criteria condition is met.
+     */
+    public boolean any_met_condition;
```

Comment:
I would advice against introducing any more public fields in the plugin

Reply:
Can I use protected or private here?
I recall I hit an issue before when not using public.